src/Kodify/BlogBundle/Controller/CommentsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Kodify\BlogBundle\Controller;
+
+use Kodify\BlogBundle\Entity\Comment;
+use Kodify\BlogBundle\Form\Type\CommentType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+
+class CommentsController extends Controller
+{
+
+    public function createAction(Request $request)
+    {
+        $form       = $this->createForm(
+            new CommentType(),
+            new Comment(),
+            [
+                'action' => $this->generateUrl('create_comment'),
+                'method' => 'POST',
+            ]
+        );
+        $parameters = [
+            'form'        => $form->createView(),
+            'breadcrumbs' => ['home' => 'Home', 'create_comment' => 'Create Comment']
+        ];
+        $form->handleRequest($request);
+        if ($form->isValid()) {
+            $comment = $form->getData();
+            $this->getDoctrine()->getManager()->persist($comment);
+            $this->getDoctrine()->getManager()->flush();
+            $parameters['message'] = 'Comment Created!';
```

Comment:
To be honest I was using the AuthorsController as a reference and I didn't think of doing it in any other way.

Checking some examples using the session flashbag I see why it could be a better option, but I'm not sure if it could be problematic if the user has cookies disabled (I guess in practice it's not really usual).

Thanks for the heads up!